lib/dal/src/audit_logging.rs

```diff
@@ -238,7 +245,7 @@ pub async fn list(ctx: &DalContext) -> Result<Vec<FrontendAuditLog>> {
         .await?;
 
     // TODO(nick): replace hard-coded max messages value with proper pagination.
-    let mut messages = consumer.fetch().max_messages(1000).messages().await?;
+    let mut messages = consumer.fetch().max_messages(10000).messages().await?;
```

Comment:
Again, calling out that this will need to be replaced soon, but I'm bumping it further for now while we test the limits.